pageobjects/loginPage.js

```diff
@@ -2,61 +2,80 @@ module.exports = {
   url: function useEnvironmentUrl() {
     return this.api.launch_url;
   },
+
   elements: {
-    loginPage: '.login',
+    landingPage: {
+      selector: ".card-pf",
+    },
+    usernameInput: {
+      selector: "#username", // ID selector for the username input field
+    },
+    passwordInput: {
+      selector: "#password", // ID selector for the password input field
+    },
+    submitBtn: {
+      selector: 'input[name="login"]', // Name selector for the submit btn field
+    },
+    rememberChk: {
+      selector: "#rememberMe", //ID selector for the rememberMe checkbox field
+    },
+    nextBtn: {
+      selector: "#kc-login", //ID selector for the rememberMe checkbox field
+    },
+    signupLink: {
+      selector: '//a[contains(text(),"Sign up")]',
+      locateStrategy: "xpath",
+    },
+    logo: {
+      selector: 'img[alt="Tidepool"]',
+    },
+    twitterLogo: {
+      selector: 'a[href="https://twitter.com/tidepool_org"]',
+    },
+    facebookLogo: {
+      selector: 'a[href="https://www.facebook.com/TidepoolOrg"]',
+    },
+    mobileLink: {
+      selector: 'a[href="http://tidepool.org/products/tidepool-mobile/"]',
+    },
+    supportLink: {
+      selector: 'a[href="http://support.tidepool.org/"]',
+    },
+    termsLink: {
+      selector: 'a[href="http://tidepool.org/legal/"]',
+    },
+    jdrfLink: {
+      selector: 'a[href="http://jdrf.org/"]',
+    },
+
+    // Add other elements of the login form here (e.g., password input, submit button, etc.)
   },
-  sections: {
-    navigation: {
-      selector: '#app',
-      elements: {
-        signupLink: 'a[href="/signup"]',
-        logo: 'img[alt="Tidepool"]',
+
+  commands: [
+    {
+      enterUsername: function (username) {
+        return this.setValue("@usernameInput", username);
```

Comment:
it's good to create commands for a series of larger, repeatable actions, but not for 1 button click. You're better off putting it in the main test if it's just one action. If logging in though is a foreseeable repeated action then you might want to create a command for the entire thing like we had before

Reply:
Yes, I only did that because it is both part of the login so I can group it with the other login actions and it is also part of the signup since a new email will go to signup page when u click next so i the event we figure our how to deal with email actions, we can still make sure the same next button will work for both login and signup route. Let me know if that makes sense